diff --git a/pdf/pdf_fapi.c b/pdf/pdf_fapi.c
index 3ea7ceaa8..d44198dee 100644
--- a/pdf/pdf_fapi.c
+++ b/pdf/pdf_fapi.c
@@ -60,7 +60,7 @@ extern const char gp_file_name_list_separator;
 extern single_glyph_list_t SingleGlyphList[];
 
 
-static void pdfi_get_server_param(gs_fapi_server * I, const char *subtype, char **server_param, int *server_param_size);
+static int pdfi_get_server_param(gs_fapi_server * I, const char *subtype, char **server_param, int *server_param_size);
 
 
 /* forward declarations for the pdfi_ff_stub definition */
@@ -1694,7 +1694,7 @@ pdfi_fapi_get_mtype_font_scaleFactor(gs_font * pfont, uint * scaleFactor)
 #endif
 
 
-static void
+static int
 pdfi_get_server_param(gs_fapi_server * I, const char *subtype,
                     char **server_param, int *server_param_size)
 {
@@ -1721,14 +1721,16 @@ pdfi_get_server_param(gs_fapi_server * I, const char *subtype,
         *server_param = NULL;
         *server_param_size = length;
     }
+
+    return 0;
 }
 #else /* UFST_BRIDGE */
-static void
+static int
 pdfi_get_server_param(gs_fapi_server * I, const char *subtype,
                     char **server_param, int *server_param_size)
 {
     *server_param = NULL;
     *server_param_size = 0;
-    return;
+    return 0;
 }
 #endif /* UFST_BRIDGE */
diff --git a/psi/zfapi.c b/psi/zfapi.c
index fc342a81e..f68f6d0ca 100644
--- a/psi/zfapi.c
+++ b/psi/zfapi.c
@@ -2358,7 +2358,7 @@ zFAPIavailable(i_ctx_t *i_ctx_p)
     return (0);
 }
 
-static void
+static int
 ps_get_server_param(gs_fapi_server *I, const byte *subtype,
                     byte **server_param, int *server_param_size)
 {
@@ -2376,6 +2376,8 @@ ps_get_server_param(gs_fapi_server *I, const byte *subtype,
             }
         }
     }
+
+    return 0;
 }
 
 static int
